Tasks/VsTest/VSTest.ps1

```diff
@@ -10,15 +10,68 @@ param(
     [string]$testRunTitle,
     [string]$platform,
     [string]$configuration,
-    [string]$publishRunAttachments
+    [string]$publishRunAttachments,
+    [string]$runInParallel
 )
 
     
-Function CmdletHasMember($memberName) {
+function CmdletHasMember($memberName) {
     $publishParameters = (gcm Publish-TestResults).Parameters.Keys.Contains($memberName) 
     return $publishParameters
 }
 
+function SetRegistryKeyForParallel() {
+    reg add HKCU\SOFTWARE\Microsoft\VisualStudio\14.0_Config\FeatureFlags\TestingTools\UnitTesting\Taef /v Value /t REG_DWORD /d 1 /f /reg:32
```

Comment:
have we tested from service and process. also do we know if this will work for hosted agents as well